friends/src/Components/FriendForm/FriendForm.js

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+
+import './FriendForm.css'
+
+class FriendForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+        name:'',
+        age:'',
+        email:''
+    }
+  }
+
+    
+  changeHandler= event=>{
+    this.setState({[event.target.name]: event.target.value})
+  }
+
+
+  submitHandler= event=>{
+    console.log(event);
+    event.preventDefault();
+    axios.post('http://localhost:5000/friends',{
+      name:this.state.name,
+      age:this.state.age,
+      email:this.state.email
+    })
+    .then(response=>console.log(response))
```

Comment:
`response.data` in the console here is the array of current friends, including the new ones. You'll want to set this new array to state in `App.js` so consider having your `axios.post` in a function sent down from state.

Also, I like that you're using your form as a a component for both your "add friend" and "update friend" routes. You could have a conditional here that checks the `path` and determines whether to send the form data to the `axios.post` function or the `axios.put` function.